package.json

```diff
@@ -28,8 +28,9 @@
     "lint": "concurrently yarn:lint:*",
     "lint:commit": "commitlint --from origin/main --to HEAD --verbose",
     "lint:es": "eslint . --ignore-path .gitignore --max-warnings 0",
+    "lint:ts": "tsc --noEmit",
     "lint:style": "stylelint '**/*.{,s}css' --ignore-path .gitignore --max-warnings 0",
-    "lint:prettier": "prettier . --check --ignore-path .gitignore",
+    "lint:prettier": "prettier . --check --ignore-unknown --ignore-path .gitignore",
```

Comment:
The thing is, e.g. https://github.com/onfido/castor/pull/683 is failing because Prettier did add support for Handlerars in [v2.3.0](https://prettier.io/blog/2021/05/09/2.3.0.html), so I believe it won't be ignore after that one is merged.

It is not merged in yet, as it is failing, and I'm not looking at that because I'm in a process of removing those files.

Reply:
Hmm good to know. We should probably run Prettier 2.3.0 on `castor-tokens`' handlebars files then.

I'll revert here.

Reply:
It is upgraded over there, however I did not find a solution, and instead just [ignored the files](https://github.com/onfido/castor-tokens/blob/cca5fcb04113cc9f1e74dcf12eded6fff59cd5bb/.prettierignore#L2).

Prettier cannot have multiple `--ignore-path`, so if custom is used, the default one (`.prettierignore`) is no longer used, and vice versa)

Reply:
We probably shouldn't ignore the file then 🤔

Reply:
I don't like how it formats it though, which then results in Sass file generated being with even stranger format...

Reply:
Ah I see. We could include a comment in the beginning of each `.hbs` file `{{! prettier-ignore }}`

Dunno if it's worth the hassle, `.prettierignore` is fine.
The only disadvantage of using a `.prettierignore` is forgetting to update it when `.gitignore` changes, which is rare.

Reply:
I tried inline ignore, it did not work... The only thing that worked was `.prettierignore`...